charts/armada-operator/templates/mutating-webhook-configuration.yaml

```diff
@@ -105,7 +105,7 @@ webhooks:
     - CREATE
     - UPDATE
     resources:
-    - lookout
+    - lookouts
```

Comment:
Oddly plural, why?

Reply:
that is by Kubernetes design,  the configuration applies for all Lookout resources, hence lookouts

Reply:
gotcha!